package.json

```diff
@@ -76,14 +76,14 @@
     "highlight.js": "^9.7.0",
     "lzma": "^2.3.2",
     "marked": "^0.3.9",
-    "fuse.js": "2.6.1",
+    "fuse.js": "^3.2.0",
```

Comment:
Any api change?

Reply:
Two options that we don't use had been removed. https://github.com/krisk/Fuse/blob/master/CHANGELOG.md